_sass/base/components/_tooltip.scss

```diff
@@ -92,3 +92,7 @@
     }
   }
 }
+
+body.dragging [role="tooltip"] {
```

Comment:
@shawnbot I'm sorry, I missed this note when I reviewed. I already merged this, derp. Ideally, though, I'd love for this to be linked to an `id` rather than a class to indicate that it is *not* for styling. If that's not possibile, then the `js-` prefix would be great.